### PR TITLE
feat(headers): add standard header

### PR DESCRIPTION
**Pull request by andreascful**
Add a standard sequence header to outgoing requests.

### CODE REVIEW

File: lib/parseOptions.js [hunk 1]
```diff
@@ -34,7 +35,8 @@ export default function parseOptions (params) {
     ...defaultOptions,
     ...configFile,
     ...params,
-    headers: params.headers || getHeadersConfig(params.header)
+    headers: addSequenceHeader(params.headers || getHeadersConfig(params.header))
+    // headers: params.headers || getHeadersConfig(params.header)
```
Comment (damienxy) [inline lib/parseOptions.js:39]: can this comment be removed?

File: test/integration/export-lib.test.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ afterAll(() => {
 })
 
 test('It should export space when used as a library', () => {
+  console.log('!!!!!!!!', spaceId)
```
Comment (damienxy) [inline test/integration/export-lib.test.js:24]: would remove this :-)

Comment (andreascful) [reply test/integration/export-lib.test.js:24]: Oh my, thank you. I was in a hurry when I pushed my changes.

**Review by marcolink — APPROVED** [review]
🚢

**Pull request merged**